args.py

```diff
@@ -25,6 +25,8 @@ def get_args():
     parser.add_argument('--synchronous', action=argparse.BooleanOptionalAction)
     parser.add_argument('--algorithm', type=str, default="dpsgd")
     parser.add_argument('--sample-size', type=int, default=10)
+    parser.add_argument('--duration', type=int, default=100)
+    parser.add_argument('--period', type=int, default=10)
```

Comment:
The period is specific to GL, isn't it? Maybe it's better to rename this parameter to `--gl-period` so it's clear that it's only used in the GL algorithm.